functional-tests/core/ft_api_test.go

```diff
@@ -27,15 +28,67 @@ var _ = Describe("Interacting with the API", func() {
 		stopHoverfly()
 	})
 
-	Context("GET /api/v2/hoverfly", func() {
+	Context("GET /api/v2/simulation", func() {
+		
+		BeforeEach(func() {
+			ImportHoverflyRecords(jsonRequestResponsePair1)
+			ImportHoverflyRecords(jsonRequestResponsePair2)
+			SetHoverflyResponseDelays("testdata/delays.json")
+		})
 
-		It("Should get all the Hoverfly config available", func() {
-			req := sling.New().Get(hoverflyAdminUrl + "/api/v2/hoverfly")
+		It("Should get all the Hoverfly simulation data in one JSON file", func() {
```

Comment:
This function is quite a beast, but I'm guessing this is simply due to the style imposed by Golang? e.g. the repeated "Expect(err).To(BeNil())"